src/toxcore/dht_new/packet.rs

```diff
@@ -761,6 +764,52 @@ impl ToBytes for NatPingResponse {
     }
 }
 
+/** Sent by both client and server, only server will respond.
+When server receives this packet it may respond with the version of the library
+plus MoTD (message of the day). The max length of MoTD is 256 bytes so the max packet
+lenght of server BootstrapInfo is 261=(1+4+256) bytes.
+
+Client must send a BootstrapInfo of exactly 78 bytes, the only 1 field is required: `packet type`
+which is filled automatically. So version may be filled with any value, so does MoTD, but
+it has to be exactly 73=(78-1-5) bytes long. The server should check that the size of the
+packet is exactly 78 bytes long (or MoTD=73 bytes filled with any values).
+
+Serialized form:
+
+Length      | Contents
+----------- | --------
+`1`         | `0xF0`
+`4`         | Version in BigEndian
+variable    | MoTD, must not longer than 256 bytes
+
+*/
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct BootstrapInfo {
+    /// The version of DHT server
+    pub version: u32,
+    /// Message of the day
+    pub motd: Vec<u8>,
+}
+
+impl ToBytes for BootstrapInfo {
+    fn to_bytes<'a>(&self, buf: (&'a mut [u8], usize)) -> Result<(&'a mut [u8], usize), GenError> {
+        do_gen!(buf,
+            gen_be_u8!(0xf0) >>
+            gen_be_u32!(self.version) >>
+            gen_slice!(self.motd.as_slice())
+        )
+    }
+}
+
+impl FromBytes for BootstrapInfo {
+    named!(from_bytes<BootstrapInfo>, do_parse!(
+        tag!(&[0xf0][..]) >>
+        version: be_u32 >>
+        motd: rest >> // TODO check motd.len <= 256 (MAX_MOTD_LENGTH)
```

Comment:
Fix todo.

Reply:
I don't know how. The solution so far is:

```
        tag!(&[0xf0][..]) >>
        version: be_u32 >>
        motd: map!(rest, |bytes| bytes.to_vec() ) >>
        verify!(value!(&motd), |motd: &Vec<_>| motd.len() <= 256) >> //MAX_MOTD_LENGTH
        (BootstrapInfo { version, motd })
```

Reply:
Thanks